src/duct/middleware/web.clj

```diff
@@ -12,23 +12,28 @@
 (def ^:private request-log-keys
   [:request-method :uri :query-string])
 
-(defn- log-request [logger request]
-  (logger/log logger :info ::request (select-keys request request-log-keys)))
+(defn- log-request
+  ([logger request]
+  (log-request logger request :info))
+  ([logger request level]
+  (logger/log logger level ::request (select-keys request request-log-keys))))
 
 (defn- log-error [logger ex]
   (logger/log logger :error ::handler-error ex))
 
 (defn wrap-log-requests
   "Log each request using the supplied logger. The logger must implement the
   duct.core.protocols/Logger protocol."
-  [handler logger]
+  ([handler logger]
+   (wrap-log-requests handler logger :info))
+  ([handler logger level]
   (fn
     ([request]
-     (log-request logger request)
+     (log-request logger request (or level :info))
```

Comment:
`level` is `:info` by default, so there's no need to check for `nil` at this point.

Reply:
Thank you.
I removed `(or ...)` and change it to `(log-request logger request level)`.